lib/components/Snackbar/SnackbarWrapper.tsx

```diff
@@ -0,0 +1,43 @@
+import { StandalonePlaceholderRegularIcon } from "@deriv/quill-icons/Standalone";
+import { Snackbar } from ".";
+import React, { useState } from "react";
+
+export const SnackbarWrapper = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const handleActionClick = () => {
+        console.log("clicked");
+        handleClose();
```

Comment:
May we just pass `handleClose` instead of `handleActionClick`?

Reply:
At the current moment yes this will work. But not when consumer plans to use action button to execute something inside the app.